deploy/9_add_bdus_xusd_pool.ts

```diff
@@ -0,0 +1,32 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types";
+import type { DeployFunction } from "hardhat-deploy/types";
+import * as constants from "../utils/Constants";
+import { formatAddress, getBdUs, getUniswapFactory } from "../utils/DeployedContractsHelpers";
+import { deployPairOracle, setupStakingContract } from "../utils/DeploymentHelpers";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  console.log("starting deployment: BDUS-XUSD");
+
+  if (hre.network.name !== "rsk" && hre.network.name !== "mainnetFork") {
+    throw Error(`Please add support for new network: ${hre.network.name}`);
+  }
+
+  const factory = await getUniswapFactory(hre);
+  const bdus = await getBdUs(hre);
+  const externalUsdStableAddress = constants.SUPPLEMENTARY_USD_STABLE_ADDRESS(hre.network.name);
```

Comment:
I like the name externalUsdStable more than supplementary in this case. maybe we should change it also on `constants`?